thefuck/rules/git_clone_missing.py

```diff
@@ -0,0 +1,42 @@
+'''
+Rule: git_clone_missing
+
+Correct missing `git clone` command when pasting a git URL
+
+```sh
+& https://github.com/nvbn/thefuck.git
+git clone https://github.com/nvbn/thefuck.git
+
+Author: Miguel Guthridge
+'''
+from urllib import parse
+
+output_contains = [
+    'not found',
+    'no such file or directory',
+    'is not recognized as',
+]
+
+
+def match(command):
+    # Ignore capitalisation in output
+    output = command.output.lower()
+    script = command.script
+    # Check for a command not found error
+    if not any([search in output for search in output_contains]):
+        return False
+    # URLs can't have spaces
+    if ' ' in command.script:
```

Comment:
Let's avoid iterating over that string once again:
```suggestion
    if len(command.script_parts) > 1:
```